dask/dataframe/io/tests/test_orc.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import shutil
+
+from dask.dataframe import read_orc
+from dask.dataframe.utils import assert_eq
+import dask.dataframe as dd
+
+pytest.importorskip('pyarrow.orc')
+url = ('https://github.com/apache/orc/blob/master/examples/'
+       'TestOrcFile.testDate1900.orc?raw=true')
```

Comment:
Perhaps we should pull this locally to a temp file location once (you can have a fixture run once per module), then use the cached version for all tests using the local filesystem.

Reply:
I had wanted to test with at least one remote data service